Makefile.toml

```
@@ -605,6 +610,32 @@ docker run --rm \
 '''
 ]
 
+[tasks.fetch-licenses]
+dependencies = ["fetch"]
+script = [
+'''
+run_fetch_licenses="
+bottlerocket-license-tool -l /tmp/Licenses.toml fetch /tmp/licenses
+"
+set +e
+if [ ! -f "${BUILDSYS_ROOT_DIR}"/Licenses.toml ] || [ ! -d "${BUILDSYS_ROOT_DIR}"/licenses ]; then
+  echo "Skipping fetching licenses since either 'Licenses.toml' or 'licenses' doesn't exist"
+  exit 0
+fi
```

Elsewhere - roles, keys - we are generally willing to create the expected storage directory if it doesn't already exist. 
```suggestion
set +e
if [ ! -f "${BUILDSYS_ROOT_DIR}"/Licenses.toml ] ; then
  echo "Not fetching licenses since 'Licenses.toml' doesn't exist"
  exit 0
fi

mkdir -p "${BUILDSYS_ROOT_DIR}/licenses"
```